javers-spring/src/main/java/org/javers/spring/aspect/SimpleDynamicPointcut.java

```diff
@@ -0,0 +1,21 @@
+package org.javers.spring.aspect;
+
+import org.springframework.aop.support.DynamicMethodMatcherPointcut;
+
+import java.lang.reflect.Method;
+
+/**
+ * @author Pawel Szymczyk
+ */
+public class SimpleDynamicPointcut extends DynamicMethodMatcherPointcut {
```

Comment:
SimpleDynamic is not very meaningfull, może coś w stylu MethodEqualsBasedMatcher?